third-party/cmake/BuildLuarocks.cmake

```diff
@@ -182,14 +182,15 @@ if(USE_BUNDLED_BUSTED)
   add_custom_target(luacheck
     DEPENDS ${LUACHECK_EXE})
 
-  set(LUV_DEPS luacheck luv-static)
+  set(LUV_DEPS luacheck luv-static lua-compat-5.3)
   if(MINGW AND CMAKE_CROSSCOMPILING)
     set(LUV_DEPS ${LUV_DEPS} libuv_host)
   endif()
   set(LUV_ARGS "CFLAGS=-O0 -g3 -fPIC")
   if(USE_BUNDLED_LIBUV)
     list(APPEND LUV_ARGS LIBUV_DIR=${HOSTDEPS_INSTALL_DIR})
   endif()
+  list(APPEND LUV_ARGS LUA_COMPAT53_INCDIR=${DEPS_BUILD_DIR}/src/lua-compat-5.3)
```

Comment:
since this is not optional(?), it could just be moved directly to the `add_custom_command` arguments.

```
  add_custom_command(OUTPUT ${HOSTDEPS_LIB_DIR}/luarocks/rocks/luv
    COMMAND ${LUAROCKS_BINARY}
    ARGS make ${LUAROCKS_BUILDARGS} ${LUV_ARGS}
      LUA_COMPAT53_INCDIR=${DEPS_BUILD_DIR}/src/lua-compat-5.3
    ...
```

Reply:
make a LUV_PRIVATE_ARGS